Makefile.core.mk

```diff
@@ -276,7 +276,7 @@ deploy-olm: bundle bundle-build bundle-push ## Build and push the operator OLM b
 
 .PHONY: undeploy
 undeploy: ## Undeploy controller from an existing cluster. Call with ignore-not-found=true to ignore resource not found errors during deletion.
-	kubectl delete istios.operator.istio.io --all --all-namespaces --wait=true
+	kubectl delete istios.operator.istio.io --all --all-namespaces --wait=true || echo "istios.operator.istio.io not found"
```

Comment:
is this supposed to be an error message, warning or just info? 
IOW, would the user (me) be wondering if the make undeploy failed?

should we just `2>/dev/null || true` to make it fail silently?

Reply:
Nevermind. I'll remove this from this PR, because other commands in this target also fail in some scenarios.